### PR TITLE
Feature hubspot contact added trigger

### LINKED ISSUE #367: HubSpot: Create or Update Contact, Add Contact to List
Actions:
- Create or Update Contact
- Add Contact to List

### PR DESCRIPTION
**Pull request by khaledmashaly**
## What does this PR do?

Add HubSpot actions:
- Create or update contact
- Add contact to list

Closes #367 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?

- [ ] Test A
- [ ] Test B

## Checklist

- I haven't performed a self-review of my own code and corrected any misspellings
- I haven't checked if my PR needs changes to the documentation
- I haven't checked if my changes generate no new warnings

### CODE REVIEW

**Comment by AbdulTheActivePiecer** [comment]
Oh wow, your file and code organization are great.
In regards to the authentication property I recommend you add all the scopes, so in the future when people who already have a connection and want to use a new action, they won't have to disconnect and connect their accounts.

**Comment by AbdulTheActivePiecer** [comment]
We should also remove Create contact, since we have create or update contact now.

File: packages/pieces/src/lib/apps/hubspot/common/props.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    }
+
+    const token = (propsValue['authentication'] as OAuth2PropertyValue).access_token;
+    const listsResponse = await hubSpotClient.lists.getStaticLists({ token });
+
+    const options = listsResponse.lists.map(list => ({
+      label: list.name,
+      value: list.listId,
+    }));
+
```
Comment (AbdulTheActivePiecer) [inline packages/pieces/src/lib/apps/hubspot/common/props.ts:50]: Please add a check for when there are no lists and add a placeholder "Please create a list"

Comment (khaledmashaly) [reply packages/pieces/src/lib/apps/hubspot/common/props.ts:50]: updated in cb72ee8

**Review by AbdulTheActivePiecer — APPROVED** [review]

**Pull request merged**